service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/generated/GeneratedOrder.java

```diff
@@ -26,4 +30,14 @@
     private final String courtName;
     @JsonSerialize(contentConverter = BasicChildConverter.class)
     private final List<Element<Child>> children;
+    private String removalReason;
+
+    @JsonIgnore
+    public boolean isRemovable() {
+        return GeneratedOrderType.fromType(type).isRemovable();
```

Comment:
I believe this can throw an exception, do we need to worry about it, or os it impossible with the current flows to get it? What about the future?

Reply:
I believe at the moment it is impossible as I have tested for all possibilities of orders, i.e. interim/final care/supervision, epo, discharge, blank. If there are more added in the future there might be issues but at the moment I don't think there should be.